app/api/order_statistics/events.py

```diff
@@ -28,9 +29,13 @@ class Meta:
 
     id = fields.Str()
     identifier = fields.Str()
-    tickets = fields.Method("tickets_count")
-    orders = fields.Method("orders_count")
-    sales = fields.Method("sales_count")
+    tickets = fields.Str()
+    orders = fields.Str()
+    sales = fields.Str()
+    # tickets = fields.Method("tickets_count")
```

Comment:
@iamareebjamal - How do i make the `tickets` variable of dynamic type like 
for when I want to work with GET call of 
`http://127.0.0.1:5000/v1/users/8/events?page[size]=0&include=order_statistics`
the tickets should be of  type `fields.Dict()`

and when I call GET `http://127.0.0.1:5000/v1/events/38/order-statistics` it should be of 
type `fields.Method("tickets_count")`

Stucked here!